editor/js/test.js

```diff
@@ -30,22 +30,22 @@ const puppeteer = require('puppeteer');
   page.on('console', msg => console.log('console>', msg.text()));
 
   // Round-trip these reactions through the DOM and compare at the server.
-  const tests = [
+  var tests = [
```

Comment:
Why not leave this as const and then create a separate validationTests (also const) array below?

Reply:
Fixed